.github/workflows/terraform-docs.yaml

```diff
@@ -0,0 +1,92 @@
+---
+'on':
+  workflow_call:
+    inputs:
```

Comment:
Small nit. Alphabetize the inputs so they are easier to find when scanning the list.